docs/operate/troubleshooting.mdx

```diff
@@ -5,6 +5,7 @@ description: "Resolving common problems with Restate deployments"
 
 import Admonition from '@theme/Admonition';
 import { Terminal } from "@site/src/components/code/terminal";
+import { Step } from "@site/src/components/Stepper";
 
 # Troubleshooting
```

Comment:
I think that Giselle is about to merge the trouble shooting page with the cluster configuration page in https://github.com/restatedev/documentation/pull/547.

Reply:
I'll merge this and help resolve any conflicts on #547, thanks for flagging it!